api/src/main/java/org/openmrs/util/Format.java

```diff
@@ -28,11 +28,18 @@ public enum FORMAT_TYPE {
 		TIME,
 		TIMESTAMP
 	}
+
 	
 	public static String formatPercentage(double pct) {
 		return NumberFormat.getPercentInstance().format(pct);
 	}
-	
+
+
+	/**
+	 * this method formats numbers to Percentage
+	 * @param  pct
+	 * @return it is static util method that allows to return null value without exception
```

Comment:
make the grammar proper

Reply:
Ok thanks @tendomart   let me improve it